src/main/java/no/bekk/bekkopen/person/FodselsnummerValidator.java

```diff
@@ -28,9 +36,16 @@
 
 	protected static final String ERROR_INVALID_INDIVIDNUMMER = "Invalid individnummer in fødselsnummer : ";
 
+  /**
+   * Truthy validation of synthetic fnr/dnr is false. You are adviced to
+   * set til value to false in test environments where you expect
+   * syntetic fnr/dnr to appair.
+   */
+	public static boolean FAIL_ON_SYNTHETIC_NUMBER = true;
```

Comment:
Jeg pleier ofte å etterstrebe at navn på sånne flagg har positiv ordlyd fordi jeg som regel synes det er enklere å lese når flaggene brukes sammen med negasjon. Jeg synes for eksempel det er kneppet enklere å tolke "not allow", enn "not fail on". Litt av samme årsak til at jeg forsøker å unngå å navngi flagg for noe med "disable" fordi jeg mentalt ser på "disable" som synonymt med "false", men dersom flagget heter "disable" så blir verdien "true" dersom noe faktisk skal være disablet.

Hva tror du om å endre flagget til dette?
```suggestion
	public static boolean ALLOW_SYNTHETIC_NUMBERS = false;
```
(med da selvfølgelig flipp på logikken de stedene flagget leses).

Da signaliseres det at syntetiske numre er ikke tillatt per default, fremfor at noe vil feile per default. Og dersom man vil endre dette så endrer man til det positive, man tillater noe, fremfor å endre til at noe ikke skal feile.

Også er det en bonus at boolean-verdien henger sammen med hvordan den er dokumentert ("Truthy validation of synthetic fnr/dnr is false.")

Reply:
Jeg skrev det først sånn selv, men så ble jeg usikker på hva andre ville føle her og snudde det. Jeg endrer gjerne. :)